scripts/bootstrap.sh

```diff
@@ -7,94 +7,159 @@ set -x
 # argument defaults
 EDX_ROLE=""
 DEPLOYMENT_ENV="dev"
-ACCESS_TOKEN=""
-OXA_TOOLS_CONFIG_VERSION="oxa/master.euc"
-OXA_TOOLS_VERSION_OVERRIDE="oxa/master.euc"
 CRON_MODE=0
 TARGET_FILE=""
-PROGRESS_FILE=""
+
+# Oxa Tools
+# Settings for the OXA-Tools public repository 
+OXA_TOOLS_PUBLIC_GITHUB_ACCOUNTNAME="Microsoft"
+OXA_TOOLS_PUBLIC_GITHUB_PROJECTNAME="oxa-tools"
+OXA_TOOLS_PUBLIC_GITHUB_PROJECTBRANCH="master"
+
+# this is the operational branch for the OXA_TOOLS public git project
+OXA_TOOLS_VERSION=""
+
+# EdX Configuration
+# There are cases where we want to override the edx-configuration repository itself
+EDX_CONFIGURATION_PUBLIC_GITHUB_ACCOUNTNAME="Microsoft"
+EDX_CONFIGURATION_PUBLIC_GITHUB_PROJECTNAME="edx-configuration"
+EDX_CONFIGURATION_PUBLIC_GITHUB_PROJECTBRANCH="oxa/master"
+
+# this is the operational branch for the EDX_CONFIGURATION public git project
+CONFIGURATION_VERSION=""
+
+# EdX Platform
+# There are cases where we want to override the edx-platform repository itself
+EDX_PLATFORM_PUBLIC_GITHUB_ACCOUNTNAME="Microsoft"
+EDX_PLATFORM_PUBLIC_GITHUB_PROJECTNAME="edx-platform"
+EDX_PLATFORM_PUBLIC_GITHUB_PROJECTBRANCH="oxa/master"
+
+# EdX Theme
+# There are cases where we want to override the edx-platform repository itself
+EDX_THEME_PUBLIC_GITHUB_ACCOUNTNAME="Microsoft"
+EDX_THEME_PUBLIC_GITHUB_PROJECTNAME="edx-theme"
+EDX_THEME_PUBLIC_GITHUB_PROJECTBRANCH="pilot"
```

Comment:
Doesn't matter. Does the branch still exist? If yes, that's fine. This value gets overwritten by the deploy-time parameters